docs/api/transactions/create.md

```diff
@@ -1,7 +1,119 @@
 ---
 id: create
-title: Create
+title: Create Transaction
 sidebar_label: Create
+keywords:
+  - OpenHIM
+  - API
+  - Transaction
+  - Routing
+  - Create
+description: Create an OpenHIM Transaction
 ---
 
-Coming soon ...
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Create Transaction via the server
+
+The OpenHIM routes through the ports `5000 (https)`, `5001 (http)` and `7786 tcp`. Any request that passes through the OpenHIM is a transaction. For the routing to happen, there has to be a channel that has the route(s) to the upstream server(s). The request has to have a path that matches the channel's `urlPattern`.
+
+To route using the OpenHIM, create a channel on the OpenHIM console or through the [API](../channels/create). If the channel is configured to be a private channel, a client will need to be [created](../clients/create). The client and the channel should be configured such that they are linked. This is done by ensuring that one of the `roles` created in client matches one of the `roles` in channel's `allow` property. The client id and password will be used for authentication. The transaction is created irregardless of whether the upstream server (mediator) being routed to exists or not.
+
+## Example routing
+
+For this example the channel that will be created is public.
+
+<Tabs
+  defaultValue="lang"
+  values={[
+    { label: 'Languages', value: 'lang' }
+  ]
+}>
+<TabItem value="lang">
+
+<Tabs
+  defaultValue="nodejs"
+  values={[
+    { label: 'NodeJS', value: 'nodejs' },
+    { label: 'Bash', value: 'bash' }
+  ]
+}>
+<TabItem value="nodejs">
+
+In this example a server that will be routed to is created and stopped soon after the routing is done. Ensure that the channel's route's url is `localhost` and the `port` matches the upstream server's `port`. The request `<PATH>` should also match the channel's `urlPattern`. If the channel was private the client's Id and password would have been passed in for the username and password.
```

Comment:
@bradsawadye I think this section could use more detail (maybe screenshots)? Or perhaps a link to the section about how to configure `channels` from the console.